ts/src/base/Exchange.ts

```diff
@@ -3187,12 +3187,12 @@ export default class Exchange {
         throw new NotSupported (this.id + ' fetchOrder() is not supported yet');
     }
 
-    async fetchOrderStatus (id: string, symbol: string = undefined, params = {}) {
+    async fetchOrderStatus (id: string, symbol: string = undefined, params = {}): Promise<Order['status']> {
```

Comment:
Can we keep `Promise<string>` for now?

Reply:
Sure. Can I add a line to remind us in the future?

`// TODO: update Promise<string> with Promise<Order['status']>`

Out of curiosity, what's blocking the change to use `Promise<Order['status']>?

Reply:
> BTW, Order looks like this already, so it already has a `string` in it.
> 
> ```
> export interface Order {
> ...    status: 'open' | 'closed' | 'canceled' | string;
> ```
> 
> Is the request for `Promise<string>` for transpiling?

@blair Yes, we have to take into consideration all languages when making a change